devices/A4988/README.md

```diff
@@ -22,21 +22,4 @@ STEP - connect to microcontroller pin
 
 ## Usage
 
-```csharp
-// Pinout for MCU please adapt depending on your MCU
-// Any regular GPIO will work
-const byte stepPin = 10;
-const byte dirPin = 11;
-const Microsteps microsteps = Microsteps.FullStep;
-const ushort fullStepsPerRotation = 200;
-TimeSpan sleepTime = TimeSpan.Zero;
-using (var motor = new A4988(stepPin, dirPin, microsteps, fullStepsPerRotation, sleepTime))
-{
-    var direction = true;
-    while (true)
-    {
-        motor.Rotate(360, direction);
-        direction = !direction;
-    }
-}
-```
\ No newline at end of file
+Please check sample project.
```

Comment:
I prefer you keep the code :-) Just copy/paste, fine. But this does appear in the nuget description. So people can copy/paste from there ;-)

Reply:
done